src/Component/Delivery/FreeDelivery.php

```diff
@@ -48,6 +48,6 @@ public function isAddressRequired()
 
     public function getName()
     {
-        return 'Free delivery';
+        return 'free_address_required';
```

Comment:
Why this name change?

Reply:
I followed the documentation https://sonata-project.org/bundles/ecommerce/master/doc/reference/bundles/delivery.html I was not able to "use an adresse" on the order delivery step